.github/workflows/release.yml

```diff
@@ -119,14 +112,13 @@ jobs:
           echo "::set-output name=RELEASE_DATE::${RELEASE_DATE}"
       -
         name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@5a54d7e660bda43b405e8463261b3d25631ffe86 # v2.7.0
```

Comment:
@dopey were you pinning to v2.7.0 of the action for a reason?
just noticed that v2.8.0 was released in October, wanted to be sure we were up to date